include/wx/textctrl.h

```diff
@@ -126,6 +126,43 @@ enum wxTextCtrlHitTestResult
 };
 // ... the character returned
 
+// ----------------------------------------------------------------------------
+// wxTextCtrl proof options object
+// Passed to ::EnableProofCheck() to configure the proofing options for
+// this control
+// ----------------------------------------------------------------------------
+class WXDLLIMPEXP_CORE wxTextProofOptions
+{
+public:
+    wxTextProofOptions(const wxString& lang = wxEmptyString)
+    : m_lang(lang)
+    {
+        m_EnableSpellCheck = true;
+        m_EnableGrammerCheck = false;
+    }
+
+    wxTextProofOptions& SpellCheck(bool enable = true)
+    {
+        m_EnableSpellCheck = enable;
+        return *this;
+    }
+
+    wxTextProofOptions& GrammerCheck(bool enable = true)
```

Comment:
Sorry, a spelling fix (sic) is needed here:
```suggestion
    wxTextProofOptions& GrammarCheck(bool enable = true)
```

Reply:
:$